src/serialize.rs

```diff
@@ -1,57 +1,74 @@
-use crate::reduction::EvalErr;
-use std::io::Cursor;
-use std::io::Read;
-use std::io::Seek;
-use std::io::Write;
-use std::io::{Error, ErrorKind, SeekFrom};
+use std::io;
+use std::io::{Cursor, ErrorKind, Read, Seek, SeekFrom};
 
 use crate::allocator::{Allocator, NodePtr, SExp};
 use crate::node::Node;
+use crate::reduction::EvalErr;
 
 const MAX_SINGLE_BYTE: u8 = 0x7f;
```

Comment:
is this constant still used anywhere now?

Reply:
I tried removing it to see and it is being used, in `impl From<EvalErr>`. It could arguably be moved to `src/reduction.rs`. Maybe I'll do this.